scalafix-tests/input/src/main/scala/test/DisableRegex.scala

```diff
@@ -0,0 +1,49 @@
+/*
+rules = Disable
+Disable.symbols = [
+  {
+    regex = "scala.collection.mutable.*"
+    message = "use immutable structures"
+  }
+]
+
+Disable.unlessInside = [
+  {
+    safeBlock = "test.DisableRegex.IO"
+    symbols = [
+      {
+        regex = {
+          includes = "java.io.*"
+          excludes = "java.io.InputStream"
+        }
+        message = "input/output are only allowed within IO block"
+      }
+    ]
+  }
+]
+ */
+package test
+
+object DisableRegex {
+  import scala.collection.mutable._ // ok
+  import scala.collection.mutable   // ok
+
+  @SuppressWarnings(Array("Disable.ListBuffer"))
+  @SuppressWarnings(Array("Disable.<init>"))
```

Comment:
I propose we either

* ignore constructors (skip when name is `<init>`), or
* give `Disable.<init>`a more helpful name such as `ListBuffer.<init>`

I vote for the first option since you can't reference a constructor without referencing the enclosing type. Having to disable twice is annoying.